test/studies/isx/isx-no-return.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: total =, bucket count =, bucket offset =, bucketize =, exchange =, count keys =
+files: isx-no-return.dat
```

Comment:
note that `files` requires to to list a file perf perfkey. If you want to use the same file for each perf key, use `repeat-files`